ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/KsqlRequest.java

```diff
@@ -114,8 +132,13 @@ public int hashCode() {
 
   @Override
   public String toString() {
+    final String sql = Objects.isNull(maskedKsql) ? ksql : maskedKsql;
+    if (Objects.isNull(maskedKsql)) {
+      LOG.warn("maskedKsql is not set, default to unmasked one for toString which "
+          + "may leak sensitive information");
```

Comment:
Thanks for adding this. One suggested addition to avoid confusion:
```suggestion
          + "may leak sensitive information. If this is seen in a test, it may be expected depending on how the KsqlRequest was created. If seen in production, this is not expected. Please file a Github issue.");
```